tools/complex_permissions_test.py

```diff
@@ -31,16 +31,16 @@ def test_inside_project_dir(self):
             ["--allow-" + self.test_type + "=" + root_path],
             [self.test_type, "package.json", "tests/subdir/config.json"])
         assert code == 0
-        assert not PROMPT_PATTERN in stderr
-        assert not PERMISSION_DENIED_PATTERN in stderr
+        assert PROMPT_PATTERN not in stderr
+        assert PERMISSION_DENIED_PATTERN not in stderr
 
     def test_outside_test_dir(self):
         code, _stdout, stderr = self._run_deno([
             "--allow-" + self.test_type + "=" + os.path.join(
                 root_path, "tests")
         ], [self.test_type, "package.json"])
         assert code == 1
-        assert not PROMPT_PATTERN in stderr
+        assert PROMPT_PATTERN not in stderr
```

Comment:
Aside.
we should probably be using `self.assertIn(PROMPT_PATTERN, stderr)`, which would give a nicer error message when this fails (it'd show stderr).

Reply:
Agreed. But let's do that in another PR.